src/index.html

```diff
@@ -6,7 +6,416 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="images/logo.png" alt="MOYO logo"
+        >
+      </a>
+      <nav>
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a href="#apple" class="header__nav-link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="header__nav-item">
```

Comment:
Add empty lines between multiline siblings.
```suggestion
          </li>
          
          <li class="header__nav-item">
```